### PR TITLE
docs: fix typo `Navigate` at `README.md`

### PR DESCRIPTION
**Pull request by paulosalvatore**
### Description

1. Problems to solve.
2. Features to add.

### How to test

1. First step.

### CODE REVIEW

**Review by moreal — COMMENTED** [review]
Thanks for your first contribution! Nice catch! 🚀 

Though I'm not a maintainer of this repository, I leave a comment because they are on holiday now. 😉 

It looks good to me but you should sign to CLA before merging this pull request. 🙏🏻 
https://github.com/planetarium/NineChronicles/pull/1097#issuecomment-1001818528

**Review by moreal — APPROVED** [review]

**Review by hskim881028 — APPROVED** [review]

**Review by sonohoshi — APPROVED** [review]

**Pull request merged**
